DTML.EduBot/Controllers/MessagesController.cs

```diff
@@ -28,6 +58,46 @@ public class MessagesController : ApiController
             return response;
         }
 
+        /// <summary>
+        /// Translate text to english and send forward
+        /// </summary>
+        /// <param name="inputText"></param>
+        /// <param name="accessToken">access token for translator API</param>
+        /// <returns></returns>
+        private async Task<string> TranslateText(string inputText, string accessToken)
```

Comment:
Would ideally move out into a seperate class. Translation in the Messages Controller, adds responsibility, but translation is actually a dependency.

In simple terms, should be a class, and can be used by the messages controller and other controllers/classes that need it.